tests/jsx-significant-space/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -90,13 +101,15 @@ before_break2 = (
     <span
       barbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbar
     />
-    {\\" \\"}foo
+    {\\" \\"}
+    foo
   </span>
 );
 
 after_break = (
   <span>
-    foo{\\" \\"}
+    foo
+    {\\" \\"}
```

Comment:
the above changes are slightly unfortunate, but the code that causes them is probably a rare edge-case. It's also probably more consistent with typical prettier behavior if anything.